tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/OrganisationsSteps.cs

```diff
@@ -150,6 +125,25 @@ public async Task GivenAnOrganisationWithNameOrganisationDoesNotExist(string org
             organisations.Should().BeNull();
         }
 
+        private object CreateOrganisation(JToken token)
```

Comment:
I believe this method could be static? It will give a very slight performance benefit & hopefully give a bit of context to other developers within the code base, indicating this is not an instance method.